packages/cache/src/cache/cache.ts

```diff
@@ -34,13 +48,20 @@ async function importCache(cacheIdentifier: TCacheIdentifiers) {
 async function getCache(
   cacheIdentifier: TCacheIdentifiers,
   adapterIdentifiers: TAdapterIdentifiers,
-  cacheKey?: string
+  cacheContext?: any
 ) {
   const cacheModule = await importCache(cacheIdentifier);
 
   const CACHE_PREFIX = getCachePrefix(adapterIdentifiers);
   const createCache = cacheModule.default;
-  const flagsCachePrefix = [CACHE_PREFIX, cacheKey].filter(Boolean).join('/');
+  let encodedCacheContext = '';
+  try {
+    encodedCacheContext = encodeCacheContext(cacheContext);
```

Comment:
At times no context may exist of the encoding fails. We then still fall back to caching more globally.